samples/request-response-mediation-interceptors/k8s-resources/api.yaml

```diff
@@ -24,6 +24,11 @@ spec:
   apiType: REST
   apiVersion: 1.0.0
   context: /interceptor/1.0.0
+  apiProperties:
```

Comment:
Use singular ones similar to the `env` defined in K8s. Refer https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/

Reply:
`apiProperties` looks fine to me